meinberlin/assets/scss/_base.scss

```diff
@@ -107,6 +107,11 @@ img {
     }
 }
 
+//stop iphone font scaling for icons
+i {
+    -webkit-text-size-adjust: 100%;
+}
+
```

Comment:
this might not work but browserstack local isn't working for me today and I can't reproduce anywhere else so if we can just merge and I can check it?

Reply:
This is for the bins that are too small? Yes, let's try!